tests/common/mod.rs

```diff
@@ -45,27 +45,24 @@ lazy_static! {
         (filename, path)
     };
     static ref CONFIG: Config = {
-        use std::{error::Error, fs::File, io::Read};
+        use std::{fs::File, io::Read};
         let mut file = File::open(IRONCORE_CONFIG_PATH.1.clone()).unwrap_or_else(|err| {
             panic!(
                 "Failed to open config file ({}) with error '{}'",
-                IRONCORE_CONFIG_PATH.0,
-                err.description()
+                IRONCORE_CONFIG_PATH.0, err
```

Comment:
Worth noting that these changes were due to description being soft deprecated, not the version upgrade.